Makefile

```diff
@@ -2,12 +2,13 @@ GOTFLATS ?=
 SHELL = /bin/bash
 GENCOMMIT = `git rev-list -1 HEAD`
 
-.PHONY: gen validate
+.PHONY: gen regen validate
 
 gen:
 	find gen/suites -maxdepth 1 -mindepth 1 -type d -print0 | xargs -I '{}' -n1 -0 bash -c 'dir="$$(basename {})" && echo "=== $${dir} ===" && cd {} && go run -ldflags "-X github.com/filecoin-project/test-vectors/gen/builders.GenscriptCommit=${GENCOMMIT}" . -o "../../../corpus/$${dir}"'
 
-
+regen:
+	find gen/suites -maxdepth 1 -mindepth 1 -type d -print0 | xargs -I '{}' -n1 -0 bash -c 'dir="$$(basename {})" && echo "=== $${dir} ===" && cd {} && go run -ldflags "-X github.com/filecoin-project/test-vectors/gen/builders.GenscriptCommit=${GENCOMMIT}" . -x -o "../../../corpus/$${dir}"'
```

Comment:
provide a `$()` or similar on gen above so that arguments to `make gen` trickle down to the go run command. Then regen can alias to `make gen -- -x` rather than duplicating the find / xargs logic.